job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -61,6 +64,14 @@ object WebApi {
     Map(ResultKey -> result)
   }
 
+  def resultToByteIterator(jobReport: Map[String, Any], result: Iterator[_]): Iterator[_] = {
+    ResultKeyStartBytes.toIterator ++
+      (jobReport.map(t => Seq(AnyJsonFormat.write(t._1).toString(),
```

Comment:
Calling AnyJsonFormat.write(jobReport) will turn it into JsObject, whereas I want to have the whole things wrapped into a JsObject including chunk encoded result, but do it in a lazy evaluated way, which is why I'm using iterators here.